Flight Software/QT RP2040/Velocity data/Pre-calibration code/Accelerometer inaccuracy and g calibration/code.py

```diff
@@ -0,0 +1,56 @@
+import board, time, busio, math, adafruit_icm20x, neopixel, csv
```

Comment:
I know that the files need to be named code.py to run, but for storage in the repo, it might be beneficial to name them what they actually do, and have them in the same folder.

What I do for CALC, is have a file named code.py, along with CALC.py. In code.py, there is only one line, `import CALC`
This will allow you to keep the file names without renaming them. `import CALC` runs whatever is in the CALC.py file.
Here is how I do it: https://github.com/besser435/Chungus-Aerospace/blob/main/Flight%20Software/CALC/code.py

This is just a suggestion though. If you would rather have named folders with code.py in each of them that is fine too.

Reply:
It'll have to be manually exited after running the calibration code anyway, so I guess that makes sense. Maybe some text files explaining everything would be good, too